dsc/docs-conceptual/dsc-3.0/concepts/configurations.md

```diff
@@ -6,7 +6,201 @@ description: >-
+
+- `$schema` (required) - Defines the canonical URI for the JSON Schema the document adheres to. DSC
+  uses this to know how to validate and interpret the document.
+- `resources` (required) - Defines the collection of resource instances the document manages.
+- `metadata` (optional) - Defines an arbitrary set of annotations for the document. DSC doesn't
+  validate this data or use it directly. The annotations can include notes like who authored the
+  document, the last time someone updated it, or any other information. DSC doesn't use the
+  annotations. The metadata is for documentation or other tools to use.
+- `parameters` (optional) - Defines a set of runtime options for the configuration. Parameters can
+  be referenced by resource instances to reduce duplicate definitions or enable dynamic values.
+  Parameters can have default values and can be set on any configuration operation.
+- `variables` (optional) - Defines a set of reusable values for the configuration. Variables can be
+  referenced by resource instances to reduce duplicate definitions.
+
 > [!NOTE]
-> The team is rewriting this document for DSCv3.
+> DSC isn't implemented to process the `parameters` and `variables` properties yet. They can be
+> defined in a document, but not referenced.
+
+## Defining a configuration document
+
+Minimally, a configuration document defines the `$schema` and `resources` properties. The
+`$schema` property must be a valid URI for DSC's configuration document schema. The `resources`
+property must define at least one DSC Resource instance.
+
+For example:
+
+```yaml
+# example.dsc.config.yaml
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+resources:
+  - name: example key value
+    type: Microsoft.Windows/Registry
+    properties:
+      keyPath:   HKCU\example\key
+      valueName: Example
+      valueData:
+        String: This is an example.
+```
+
+The example document declares a single resource instance called `example key value`. The instance
+uses the `Microsoft.Windows/Registry` resource to declare that:
+
+- The `example\key` registry key should exist in the system's current user hive.
+- The `example\key` registry key should have a value called `Example`.
+- The `Example` value should be the string `This is an example`.
+
+The example document is _declarative_. It describes the desired state, not how to put the system
+into that state. It relies on the `Microsoft.Windows/Registry` resource to handle getting, testing,
+and setting the state of the registry key and value.
+
+### Defining resource instances
+
+As shown in the prior example, configuration documents include a collection of resource instances.
+Together, the instances describe a system's desired state. A configuration document can include any
+number of resource instances. The instances can be of any available DSC Resource.
```

Comment:
```suggestion
Together, the instances describe the desired state of a system. A configuration document can include any
number of resource instances.
```

I am not sure what that last sentence (removed) was supposed to mean.

Reply:
Trying to clarify that you're not limited to defining instances of a single resource type at a time - agree with cutting though.